src/components/molecules/TextDisplay.tsx

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
```

Comment:
Excellent with a TextDisplay component. However, since it is "dumb" (no local state or logics nor complex usage of other components, I would like to see this component to be within the "atoms" folder :)